From fed3a4166387e1b9a4a51e1b88eee178095cd707 Mon Sep 17 00:00:00 2001
From: Craig Kewley <craigx.kewley@intel.com>
Date: Fri, 18 Aug 2017 09:33:34 +0100
Subject: [PATCH 59/98] Revert "ASoC: Intel: Boards: remove dependency on
 TDF8532 codec"

This reverts commit 227b50e2735733fae1d3dc2770d360074c3b3637.

Conflicts:
	sound/soc/intel/Kconfig
---
 sound/soc/intel/boards/Kconfig     | 2 +-
 sound/soc/intel/boards/bxt_gpmrb.c | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/sound/soc/intel/boards/Kconfig b/sound/soc/intel/boards/Kconfig
index 80f4380..a0322a2 100644
--- a/sound/soc/intel/boards/Kconfig
+++ b/sound/soc/intel/boards/Kconfig
@@ -388,7 +388,7 @@ config SND_SOC_INTEL_BXTP_IVI_GENERIC_MACH
 	   If unsure select "N".
 
 config SND_SOC_INTEL_BXT_GPMRB_MACH
-	tristate "ASoC Audio driver for BXT IVI GP MRB with Dummy codec"
+	tristate "ASoC Audio driver for BXT IVI GP MRB"
 	select SND_SOC_INTEL_SST
 	select SND_SOC_TDF8532
 	select SND_SOC_COMPRESS
diff --git a/sound/soc/intel/boards/bxt_gpmrb.c b/sound/soc/intel/boards/bxt_gpmrb.c
index 42a36cc..83fbd94 100644
--- a/sound/soc/intel/boards/bxt_gpmrb.c
+++ b/sound/soc/intel/boards/bxt_gpmrb.c
@@ -492,8 +492,8 @@ static int broxton_gpmrb_tdf8532_startup(struct snd_pcm_substream *substream)
 		.name = "SSP4-Codec",
 		.be_id = 0,
 		.cpu_dai_name = "SSP4 Pin",
-		.codec_name = "snd-soc-dummy",
-		.codec_dai_name = "snd-soc-dummy-dai",
+		.codec_name = "i2c-INT34C3:00",
+		.codec_dai_name = "tdf8532-hifi",
 		.platform_name = "0000:00:0e.0",
 		.dpcm_playback = 1,
 		.no_pcm = 1,
-- 
1.9.1

